src/sources/host_metrics/cgroups.rs

```diff
@@ -25,13 +25,21 @@ pub(crate) struct CGroupsConfig {
     /// The number of levels of the cgroups hierarchy for which to report metrics.
     ///
     /// A value of `1` means just the root or named cgroup.
-    #[derivative(Default(value = "100"))]
+    #[derivative(Default(value = "default_levels()"))]
+    #[serde(default = "default_levels")]
+    #[configurable(metadata(docs::examples = 1))]
+    #[configurable(metadata(docs::examples = 3))]
     levels: usize,
 
     /// The base cgroup name to provide metrics for.
+    #[configurable(metadata(docs::examples = "/"))]
+    #[configurable(metadata(docs::examples = "system.slice/snapd.service"))]
     pub(super) base: Option<PathBuf>,
 
-    /// Lists of group name patterns to include or exclude.
+    /// Lists of cgroup name patterns to include or exclude in gathering
+    /// usage metrics.
+    #[configurable(metadata(docs::examples = "example_cgroups()"))]
+    #[serde(default = "default_all_devices")]
     groups: FilterList,
 
     /// Base cgroup directory, for testing use only
```

Comment:
This needs a `hidden` attribute? For testing only I'd assume we don't want it rendered on the site.

Reply:
<img width="853" alt="Screenshot 2023-01-17 at 3 45 49 PM" src="https://user-images.githubusercontent.com/42553923/213036859-4f3a2ab0-de62-4aaa-b63f-48022083490f.png">

Looks like we do want it to show up on the site (from vector.dev).

Reply:
Sorry - not that section, this one: https://deploy-preview-15956--vector-project.netlify.app/docs/reference/configuration/sources/host_metrics/#cgroups.base_dir

<img width="662" alt="image" src="https://user-images.githubusercontent.com/11903873/213204103-3cf90324-b7fb-4401-948a-394aebc6f0f7.png">

Reply:
Got it, thanks.